lxdock/conf/config.py

```diff
@@ -115,6 +153,27 @@ def _get_container_config_dict(self, container_dict):
         del container_config['containers']
         return container_config
 
+    def _get_dotenv_items(self, env_filepath):
```

Comment:
... but then it's probably a dead end. After all, they developed a full-featured library at https://github.com/bkeepers/dotenv . It seems like we have 2 basic choices:

1. Develop a full featured dotenv-parsing library. A quick google search only reveal libraries that don't go much further than your parsing routine.
2. Accept and document sub-standard parsing.

The problem with going with the second option will be that this routine risk being gradually "improved" through various PR to make it more "bash-like" and will end up being bigger than we'd like.